equinox/nn/pool.py

```diff
@@ -256,3 +256,35 @@ def __init__(
             padding=padding,
             **kwargs,
         )
+
+
+class AdaptiveAvgPool1d(Module):
+    """Adaptive 1D downsampling for a target shape."""
+
+    target_size: int = static_field()
+
+    def __init__(self, target_size: int):
+        """**Arguments:**
+        - `target_size`: The target output size.
```

Comment:
Nit: this will need a blank line between these two lines, in order to render correctly in the docs.